styles.css

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
In tegenstelling tot veel anderen studenten heb je relatief weinig css (wat goed is!) 👍